src/main/java/com/faforever/api/config/FafApiProperties.java

```diff
@@ -113,6 +115,15 @@
     private String downloadUrlFormat;
   }
 
+  @Data
+  public static class Avatar {
+    private String downloadUrlBase;
+    private List<String> allowedFileExtensions;
+    private Integer maxSizeBytes;
```

Comment:
`allowedFileExtensions` and `maxSizeBytes` should have default values (hint: Use `Arrays.asList()`)